database/migrations/2023_11_06_084718_event_page_update_title_length.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\EventPageUpdate;
+
+class EventPageUpdateTitleLength extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('event_page_updates', function (Blueprint $table) {
+            $table->text('title')->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        EventPageUpdate::query()->get()->each(function (EventPageUpdate $eventPageUpdate) {
```

Comment:
This is probably a _very_ inefficient way of doing, but considering we will likely never run it, I went for something that is easy to understand.